deploy/csv/marketplace.csv.yaml

```diff
@@ -0,0 +1,123 @@
+  selector:
+    matchLabels:
+      name: marketplace-operator
+  links:
+  - name: Markplace Operator Source Code
+    url: https://github.com/operator-framework/operator-marketplace
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
+      - serviceAccountName: default
+        rules:
+        - apiGroups:
+          - marketplace.redhat.com
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - configmaps
+          verbs:
+          - "*"
+        - apiGroups:
+          - operators.coreos.com
+          resources:
+          - catalogsources
+          verbs:
+          - "*"
+      deployments:
+      - name: marketplace-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: marketplace-operator
+          template:
+            metadata:
+              name: marketplace-operator
+              labels:
+                name: marketplace-operator
+            spec:
+              containers:
+                - name: marketplace-operator
+                  image: quay.io/redhat/marketplace
+                  ports:
+                  - containerPort: 60000
+                    name: metrics
+                  command:
+                  - marketplace-operator
+                  imagePullPolicy: Always
+                  env:
+                    - name: WATCH_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.namespace
+                    - name: OPERATOR_NAME
+                      value: "marketplace-operator"
+customresourcedefinitions:
+    owned:
+    - name: operatorsources.marketplace.redhat.com
+      version: v1alpha1
+      kind: OperatorSource
+      displayName: Operator Source
+      description: Represents an Operator Source.
+      specDescriptors: 
+        - description: The type of the operator source.
+          displayName: Type
+          path: type
+        - description: Points to the remote app registry server from where operator manifests can be fetched.
+          displayName: Endpoint
+          path: endpoint
+        - description: |-
+            The namespace in app registry.
+            Only operator manifests under this namespace will be visible.
+            Please note that this is not a k8s namespace.
+          displayName: Registry Namespace
+          path: registryNamespace
+      statusDescriptors:
+      - description: The Current phase of the OperatorSource object.
+        displayName: Current Phase
+        path: currentPhase
+        x-descriptors:
+          - 'urn:alm:descriptor:io.kubernetes.phase'
+    - name: catalogsourceconfigs.marketplace.redhat.com
+      version: v1alpha1
+      kind: CatalogSourceConfig
+      displayName: Catalog Source Config
+      description: Represents an Catalog Source Config.
```

Comment:
`CatalogSource`

Reply:
I think it is better to leave it as `Represents a CatalogSourceConfig` because the description is for the object. Or it can be `Represents a CatalogSourceConfig object which is used to configure a CatalogSource`.

Reply:
Should be all set. I went with the more detailed response.

Reply:
@aravindhp could you please resolve this conversation if the latest commit meets you're expectations. Thanks!